packages/docs/docs/features/network-inspector.md

```diff
@@ -58,11 +60,12 @@ The Network Inspector displays a log of all network requests made by your applic
 - **Name** - the segment of the path of the URL request.
 - **Status** - a HTTP status code returned by the server.
 - **Method** - a HTTP request method used (e.g. GET, POST, PUT, DELETE, etc.).
-- **Type** - a MIME type of the requested resource. Currently, only `XHR` type is supported.
+- **Type** - a resource type as it was perceived by the rendering engine.
```

Comment:
I don't understand what this means. What does "rendering engine" refer to here?

I think it's ok to be more vague with this, like:

```suggestion
- **Type** - type of the received response.
```